src/ProgramBlockEditor.js

```diff
@@ -20,6 +20,7 @@ import './ProgramBlockEditor.scss';
 type ProgramBlockEditorProps = {
     intl: any,
     activeProgramStepNum: ?number,
+    actionButtonDisabled: boolean,
```

Comment:
Rather than a new property called `actionButtonDisabled` let's make the property the same name as the App state value: `modalIsShowing`. This will communicate that it is the ProgramBlockEditor's responsibility to render itself appropriately when a modal is being displayed.